tests/UI/Admin_spec.js

```diff
@@ -40,10 +40,12 @@ describe("LoginLdap_Admin", function () {
         expect.screenshot('admin_page').to.be.captureSelector('#content', function (page) {
             page.load(ldapAdminUrl);
 
+              page.sendKeys('input#memberOfField', 'a');
             page.sendKeys('input#memberOf', 'a');
             page.sendKeys('input#filter', 'a');
 
             page.evaluate(function () {
+                $('input#MemberOfFIeld').val('memberOf').trigger('input');
```

Comment:
The changes to this file are unnecessary. If the default memberOf field is 'memberOf', setting it here will have no effect. Can you remove both of the changes to this file?